spec/jobs/central_mail/submit_saved_claim_job_spec.rb

```diff
@@ -20,9 +25,9 @@
       expect(job).to receive(:generate_metadata).and_return(
         metadata: {}
       )
-      expect_any_instance_of(CentralMail::Service).to receive(:upload).with(
+      expect(central_mail_service_stub).to receive(:upload).with({
```

Comment:
If we don't enclose this argument in curly braces, rails interprets it as 'keyword arguments' instead of 'options hash, ie.
```
         expected: ({ "metadata" => "{\"metadata\":{}}", "document" => "faraday1", "attachment1" => "faraday1" }) (keyword arguments)
              got: ({ "metadata" => "{\"metadata\":{}}", "document" => "faraday1", "attachment1" => "faraday1" }) (options hash)
```

This isn't an especially helpful in any case, but in this case it's explicitly incorrect.  `CentralMail::Service#upload` expects a hash in the actual code, so making this change to make a spec pass also makes the test _more correct_ ¯\_(ツ)_/¯